portal/src/main/scala/cromwell/pipeline/controller/ProjectController.scala

```diff
@@ -0,0 +1,40 @@
+package cromwell.pipeline.controller
+
+import akka.http.scaladsl.model.StatusCodes
+import akka.http.scaladsl.server.Route
+import akka.http.scaladsl.server.Directives._
+import cromwell.pipeline.datastorage.dto.project.ProjectAdditionRequest
+import cromwell.pipeline.service.ProjectService
+import cromwell.pipeline.utils.auth.AccessTokenContent
+import de.heikoseeberger.akkahttpplayjson.PlayJsonSupport._
+
+import scala.concurrent.ExecutionContext
+import scala.util.{ Failure, Success }
+
+class ProjectController(projectService: ProjectService)(
+  implicit executionContext: ExecutionContext
+) {
+
+  val route: AccessTokenContent => Route = accessToken =>
```

Comment:
There is security hole, every user can amend all projects, that's not good. You should handle accessToken in service layer.

Reply:
I'll fix it

Reply:
Done